pkg/rbac/userData_test.go

```diff
@@ -114,7 +114,10 @@ func Test_getNamespaces_usingCache(t *testing.T) {
 	//mock cache for cluster-scoped resouces to get all namespaces:
 	mock_cache.shared.namespaces = append(namespaces, "some-namespace")
 	//mock cache for managed clusters
-	mock_cache.shared.managedClusters = append(managedclusters, "some-namespace", "some-nonmatching-namespace")
+	managedclusters := make(map[string]struct{})
+	managedclusters["some-namespace"] = struct{}{}
+	managedclusters["some-nonmatching-namespace"] = struct{}{}
+	mock_cache.shared.managedClusters = managedclusters
```

Comment:
Suggestion: Could be compressed to a single line.
```suggestion
	mock_cache.shared.managedClusters = map[string]struct{} {"some-namespace": struct{}{}, "some-nonmatching-namespace": struct{}{} }
```

Suggestions like this can be personal preference so take it as optional. 😃

Reply:
Updated